### DISCUSSION TITLE
ProxSuite 0.6.4

### DISCUSSION THREAD

**Discussion by jcarpent** [Simple-Robotics/proxsuite#305] [Announcements]
### Fixed

* Changed `primal_infeasibility_solving` to `False` for feasible QPs ([#302](https://github.com/Simple-Robotics/proxsuite/pull/302))

**Full Changelog**: https://github.com/Simple-Robotics/proxsuite/compare/v0.6.3...v0.6.4

<hr /><em>This discussion was created from the release <a href='https://github.com/Simple-Robotics/proxsuite/releases/tag/v0.6.4'>ProxSuite 0.6.4</a>.</em>

**Comment by nim65s** [comment]
Hello !

This released wasn't merged in main, and not uploaded on PyPI.
Should I handle this ?
